test/common/index.js

```diff
@@ -83,7 +83,8 @@ function parseTestFlags(filename = process.argv[1]) {
   return source
     .substring(flagStart, flagEnd)
     .replace(/_/g, '-')
-    .split(' ');
+    .split(/\s+/)
+    .filter(Boolean);
```

Comment:
I don't think we need the `.filter`  given that the regex is greedy.
```suggestion
    .split(/\s+/);
```

Reply:
without the filter, we end-up with empty strings in some of the cases